src/main/java/website/frontrow/game/Game.java

```diff
@@ -214,6 +233,36 @@ public void levelWon()
     @Override
     public void levelLost()
     {
+        gameOver();
         stop();
     }
+    
+    /**
+     * Returns whether or not all players are dead.
+     * @return b boolean
+     */
+    public boolean playersDead()
+    {
+    	if(!this.isRunning())
+    	{
+    		return false;
+    	}
+    	boolean b = true;
+    	for(Player p : this.players)
+    	{
+    		if (p.playerAlive())
+    		{
+    			b = false;
+    		}
+    	}
+    	return b;
+    }
+    
+    /**
+     * Adds Game Over message to log.
+     */
+    public void gameOver()
+    {
+    	addToLog("GAME OVER");
```

Comment:
Please, use the logger format of: [CLASS_NAME]\t[Additional Note if applicable]\t<Message>